.gitignore

```diff
@@ -7,9 +7,13 @@ node_modules/
 # Where ENV secrets are stored. refer to the "Note" at the link below for details on the use of this file
 # https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables
 .env.local
+.env.development
```

Comment:
why'd you add this file? what's in it?

Reply:
.env.development is our standard env file for all apps. it means we can use stack_car on it. .env.local is similar in purpose, but just trying to keep things consistant